src/librbd/exclusive_lock/PostAcquireRequest.cc

```diff
@@ -172,11 +180,48 @@ void PostAcquireRequest<I>::handle_allocate_journal_tag(int r) {
     return;
   }
 
+#if defined(WITH_RBD_RWL)
```

Comment:
Nit: this `#if` section is particularly hard to read since it injects across the middle of multiple functions.

Reply:
Moved it to rwl/InitRequest.